parts/k8s/cloud-init/artifacts/cse_config.sh

```diff
@@ -46,52 +47,47 @@ configureEtcdUser(){
   id etcd
 }
 configureSecrets(){
-  APISERVER_PRIVATE_KEY_PATH="/etc/kubernetes/certs/apiserver.key"
-  touch "${APISERVER_PRIVATE_KEY_PATH}"
-  CA_PRIVATE_KEY_PATH="/etc/kubernetes/certs/ca.key"
-  touch "${CA_PRIVATE_KEY_PATH}"
-  ETCD_SERVER_PRIVATE_KEY_PATH="/etc/kubernetes/certs/etcdserver.key"
-  touch "${ETCD_SERVER_PRIVATE_KEY_PATH}"
+  local apiserver_key="/etc/kubernetes/certs/apiserver.key" ca_key="/etc/kubernetes/certs/ca.key" etcdserver_key="/etc/kubernetes/certs/etcdserver.key"
+  touch "${apiserver_key}"
```

Comment:
`touch` takes multiple arguments, so these three commands could be a one-liner for a savings of...drumroll...12 bytes.